doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
++--------------+---------------------------------------------------------------+
+| State        | Work in progress to support all the functionality from        | 
+|              | plutus-ledger-constraints.                                    |
++--------------+---------------------------------------------------------------+
+
+6. Plutus chain index
+---------------------------------
+
+`Plutus chain index <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-chain-index-core>`_ 
+is an application for indexing useful data from the Cardano blockchain that is to be 
+used in the Contract Monad. The main design is for indexing information related to the 
+UTxO set in order to keep the amount of indexed information proportional to the 
+UTxO set. The component is not meant to index historical information, contrary to 
+the purpose of cardano-db-sync. 
+
++--------------+---------------------------------------------------------------+
+| Plutus chain index                                                           |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-chain-index-core``                       |
+|              | ``plutus-apps/plutus-chain-index``                            |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-script-utils                                           |
++--------------+---------------------------------------------------------------+
+| State        | In the process of being deprecated in favor of Marconi.       | 
++--------------+---------------------------------------------------------------+
+
+7. Contract monad (aka contract API, plutus contract)
+-----------------------------------------------------------
+
+`Contract monad <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library for writing Plutus contracts. It provides the Contract Monad interface 
+for writing the off-chain part of a Plutus application that is to be interpreted 
+by an emulator or a plutus application backend (PAB). 
+
++--------------+---------------------------------------------------------------+
+| Contract monad                                                               |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-ledger-constraints, plutus-chain-index-core            |
++--------------+---------------------------------------------------------------+
+| State        | Maintained but not extended. Unstable interface.              | 
```

Comment:
This sounds quite negative, as if it is not intended to be used. Maybe "Maintained with a stable set of features"?

Reply:
Yeah, I was a bit confused too by the wording "Maintained but not extended. Unstable interface" Why is it unstable if we are not going to change it?

Reply:
I agree that it is sending mixed messages and could be confusing and potentially discouraging. I'll talk with Konstantinos about a more effective way to communicate about this issue.

Reply:
After discussing with Konstantinos, we decided to remove the "State" throughout this page because it's not very helpful information for readers. Instead, we are characterizing the overall Tools project as "in development." See Draft 2.